lib/glueby/contract/payment.rb

```diff
@@ -12,41 +12,41 @@ module Contract
     #            Glueby::Wallet.create
     #
     # Balance of sender and reciever before send
-    # sender.internal_wallet.balance
+    # sender.balances[""]
     # => 100_000(tapyrus)
-    # reciever.internal_wallet.balance
+    # reciever.balances[""]
```

Comment:
```suggestion
    # receiver.balances[""]
```
also following same.

Reply:
Thank you. I fix it.

Reply:
Thank you. I fix it.